### DISCUSSION TITLE
maxs19.fun: detection

### DISCUSSION THREAD

**Discussion by Gali-V** [uBlockOrigin/uAssets#23137] [General]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://maxs19.fun/watchfree/d42rmjfkqhx3/45vx9j4q61zb/enQ5aUtWck5xMEhxUnE1emFvNzhBZz09`

### Category

detection

### Description

detection 

### Other extensions used

no

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.56.0
Chromium: 123
filterset (summary):
 network: 129842
 cosmetic: 52542
 scriptlet: 20323
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ITA-0: 8382-18, 5m
  easylist: 84586-146, 5m
  easyprivacy: 50416-12, 5m
  plowe-0: 3762-1183, 5m
  ublock-badware: 8061-140, 5m
  ublock-filters: 37405-370, 5m
  ublock-privacy: 781-5, 5m
  ublock-quick-fixes: 193-6, 5m
  ublock-unbreak: 2250-33, 5m
  urlhaus-1: 8940-5, 5m
filterset (user): [empty]
switchRuleset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 2893 ms
 maxAssetCacheWait: 24 ms
 cacheBackend: browser.storage.local
popupPanel:
 blocked: 12
 network:
  afodreet.net: 1
  celeritascdn.com: 1
  esoussatsie.xyz: 2
  geiybze.com: 1
  googletagmanager.com: 2
  groorsoa.net: 2
  host-cdn.net: 1
  intellipopup.com: 1
  maxstream.video: 1
 extended:
  ###block-boxes-taboola
  ##+js(no-setTimeout-if, alert)
  ##+js(adjust-setTimeout, .fadeIn(), 3000)
  ##+js(no-window-open-if)
```
</details>

**Comment by mapx-** [comment]
error code 1005 => vpn can not be used, we cannot investigate the issue

**Reply by mapx-** [reply]
converted in discussion, if someone can test it using a native italian ip
